test/acceptance/asset-discovery.test.js

```diff
@@ -0,0 +1,27 @@
+import expect from 'expect'
+import {
+  run,
+  setupApiProxy,
+  setupDummyApp
+} from './helpers'
+
+describe('Asset discovery', () => {
+  let proxy = setupApiProxy()
+  let dummy = setupDummyApp()
+
+  it('does not create snapshots while stopping a build', async () => {
+    let [stdout] = await run('percy exec -- node ./test/acceptance/dummy/snapshot-error.js');
+
+    let finalizeReqDate = proxy.requests['/builds/123/finalize'][0].requestDate;
+    let lastSnapshotReqDate = proxy.requests['/builds/123/snapshots'].pop().requestDate;
```

Comment:
I dislike this mutation, but the proxy is reset before each test, so my disdain isn't valid.

Reply:
Its valid, I just wanted a clean way to get the last item. I'll not mutate